README.md

```diff
@@ -1,19 +1,40 @@
 # 3Box
-User-Managed Application Data for Web3.
+3Box is a distributed database that supports public and private data for Ethereum users. All data is publicly available, but private data can only be decrypted by those to whom the user has provided permission. 
 
-[3box documents folder](https://drive.google.com/drive/folders/16lZWMVFLKLk2nAZJQ7xQyzHKZzK734Ov?usp=sharing)
+## Quick Links
 
-## Technical specification
-3box consists of two major components `3box-root-hash-tracker` which is a server component and `3box-js` which is a client side library. Together they are used to keep track of user data stored in `orbit-db` instances. Below are detailed specifications of how they are implemented.
+### For Ethereum Users (Coming Soon)
+👤 Create an Ethereum Profile at 3box.io.
 
-### 3box-root-hash-tracker
-The root hash tracker is a REST-API that is used to associate an ethereum address with an ipfs hash.
+### For Ethereum Dapp Developers (Coming soon)
+👩‍💻 Want to integrate 3Box data? [Explore 3box-js](https://www.github.com/uport-project/3box-js)
 
-[*3box-root-hash-tracker specification*](./3BOX-ROOT-HASH-TRACKER.md)
+### For Our Community
+👍 Want to collaborate on code? Submit an issue or a PR!  
```

Comment:
We should probably have a "how to contribute" guide before soliciting work from others.

Reply:
Is that a blocker, or something we should develop a simple version of soon? I don't imagine a flood of PRs all of a sudden. But maybe? @christianlundkvist

Reply:
https://github.com/uport-project/3box/issues/136

^ @christianlundkvist added an issue for the how to contribute guide, but keeping this here for now. I don't expect a bunch of submission just yet and if we do that's great and we'll prioritize the guide